evaluation/benchmarks/poets/8.2_2.sh

```diff
@@ -4,4 +4,4 @@
 
 # Bigrams that appear twice
 IN=${IN:-$PASH_TOP/evaluation/benchmarks/poets/input/pg/}
-cat ${IN}* | awk "\$1 == 2 {print \$2, \$3}" 
+ls ${IN} | sed "s;^;$IN;"| xargs cat | awk "\$1 == 2 {print \$2, \$3}" 
```

Comment:
Are we sure this script is correct? Does this really print bigrams that appear twice?